lms/djangoapps/verify_student/management/commands/tests/test_populate_expiry_date.py

```diff
@@ -31,78 +32,27 @@ def create_and_submit(self, user):
         attempt.upload_photo_id_image("More Fake Data")
         attempt.mark_ready()
         attempt.submit()
+        attempt.expiry_date = now() + timedelta(days=FAKE_SETTINGS["DAYS_GOOD_FOR"])
```

Comment:
Why setting this field to a meaningful value? Why not just set it to `None`?

Reply:
Here the date itself doesn't really matter, but since I'm changing the logic so that it's filtering for entries with an `expiry_date`, I want that field to have some value.